lib/Fhp/Segment/WPD/HIWPDSv5.php

```diff
@@ -0,0 +1,27 @@
+<?php /** @noinspection PhpUnused */
+
+namespace Fhp\Segment\WPD;
+
+
+
+use Fhp\Segment\BaseGeschaeftsvorfallparameter;
+use Fhp\Segment\BaseGeschaeftsvorfallparameterOld;
+
+/**
+ * Segment: Saldenabfrage Parameter (Version 5)
+ *
+ * @link https://www.hbci-zka.de/dokumente/spezifikation_deutsch/archiv/HBCI_V2.x_FV.zip
+ * File: HBCI22 Final.pdf
+ * Section: VII.2.2 c)
+ */
+ 
+class HIWPDSv5 extends BaseGeschaeftsvorfallparameterOld implements HIWPDS
```

Comment:
I believe this uses BaseGeschaeftsvorfallparameter and not the "Old" variant because it's in the FinTS 3.0 specification.

Reply:
I thought so, but comdirect seems to use the old one. Attribute "Sicherheitsklasse" was not submitted. It is specified in BaseGeschaeftsvorfallparameter but not in BaseGeschaeftsvorfallparameterOld, so I used it.